### PR TITLE
docs: Rewrite README & in-app instructions

### PR DESCRIPTION
**Pull request by machour**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | 
| Fixed issues  | #60 #53

I've fixed the README instruction to be more accurate and helpful.

I'm also testing available classes and displaying instructive error messages to help the User understand what's going on.

Couldn't resist adding 4 lines of CSS in this PR, considering that the broken design isn't helping either :)

screen shot 2018-12-17 at 8 38 04 pm

### CODE REVIEW

File: src/views/layouts/main.php [hunk 1]
```diff
@@ -22,6 +22,20 @@
     <?php $this->registerCsrfMetaTags() ?>
     <title><?= Html::encode($this->title) ?></title>
     <?php $this->head() ?>
+    <style>
+        ul.navbar-nav li, ul.breadcrumb li {
```
Comment (schmunk42) [inline src/views/layouts/main.php]: How about putting that into an asset bundle? `src/assets` with its CSS files also located there.

Comment (machour) [reply src/views/layouts/main.php]: Definitely. I just realized that there is already an outdated site.css file shipped. Should I update it with these (and maybe a little more ui-:heart:)?

Comment (schmunk42) [reply src/views/layouts/main.php]: Yes, would be great to update it.

File: README.md [hunk 2]
```diff
@@ -61,7 +61,8 @@ CONFIGURATION
 You should recreate configuration files as needed in your root `config/` directory.
 
 Make sure they are listed in the `extra.config-plugin` entry of your `composer.json` file in order for them
-to be discovered, and remember to run `composer install` every time you change a configuration value.
+to be discovered, and remember to run `composer dump-autoload` (or `composer du`) every time you change a 
+configuration value.
```
Comment (schmunk42) [inline README.md:65]: Might also be good to add an auto-refresh for `YII_ENV_DEV`, see also https://github.com/hiqdev/composer-config-plugin#refreshing-config

File: src/views/layouts/main.php (hunk 1 above)
Comment (machour) [reply src/views/layouts/main.php]: It looks like this now:

screen shot 2018-12-17 at 11 56 48 pm

File: README.md (hunk 2 above)
Comment (machour) [reply README.md:65]: Added a link to this section in the README for now.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**